Assets/Scripts/UI/ObjectiveList/ObjectiveList.cs

```diff
@@ -22,12 +21,16 @@ void Start()
 
     void Update()
     {
-        if (Input.GetKeyDown(KeyCode.Space) && !root.GetBool("slideOut"))
+        print(Input.GetAxis(Constants.INPUT_AXIS_LEFT_DPAD));
+        if (((Input.GetAxis(Constants.INPUT_AXIS_LEFT_DPAD) != 0) || Input.GetKeyDown("j")) && !root.GetBool(Constants.ANIMATION_OBJECTIVELIST_SLIDE))

```

Comment:
I think this will make so both left and right dpad work... not sure if positive values are left or right, but id check for that and only allow that one side!

Reply:
Okay, I just followed this now https://answers.unity.com/questions/1536528/d-pad-as-button-instead-of-axis.html 
So let me know if it works! :)